### DISCUSSION TITLE
Front camera

### DISCUSSION THREAD

**Discussion by Ysrbolles** [google/model-viewer#4017] [Q&A]
Is it possible to select the front camera in AR mode and take a selfie  with the 3D project ?

**Comment by elalish** [comment]
No.

**Reply by Ysrbolles** [reply]
@elalish  Is there any feature about this part coming next?

**Reply by elalish** [reply]
No, because WebXR doesn't support the front camera. The other AR modes are separate apps we don't control, so it's not currently possible to implement. Also, you probably wouldn't place the object on the floor with the front camera, so a whole new UX would probably have to be designed.
